packages/replay-next/components/inspector/protocolValueToText.ts

```diff
@@ -59,7 +58,7 @@ async function protocolValueToTextHelper(
     } else {
       objectIdSet.add(objectId);
 
-      const object = await getObjectWithPreviewHelper(client, pauseId, objectId, true);
+      const object = await objectCache.readAsync(client, pauseId, objectId, "full");
```

Comment:
:speech_balloon: Thanks for turning this into an enum! It's now much more clear what the point of that arg is :)

Reply:
Agreed! 😁